src/pages/partners/affiliate-ib/_minimum-volume.tsx

```diff
@@ -67,6 +63,19 @@ const StyledHeader = styled(Header)`
         margin: 0 auto 16px;
     }
 `
+
+const SubtitleHeader = styled(Header)`
+    line-height: 36px;
+
+    @media ${device.laptopL} {
+        width: 100%;
+    }
+    @media ${device.tabletL} {
+        line-height: 24px;
+        font-size: 16px;
```

Comment:
This property exist too
Better approach it find the figma's design, check px of title and pass type equal for desktop 
<img width="654" alt="Screenshot 2023-03-22 at 09 18 50" src="https://user-images.githubusercontent.com/103182473/226818488-eb0bb106-a25d-4a61-81e3-5b227b46de79.png">